lib/async_hooks.js

```diff
@@ -29,6 +30,7 @@ const internal_async_hooks = require('internal/async_hooks');
 // resource gets gced.
 const { registerDestroyHook } = internal_async_hooks;
 const {
+  asyncWrap,
```

Comment:
Why expose this when asyncWrapProviders is also exposed? Also this one would not be frozen while the other is. 🤔

Reply:
Nevermind, just read it wrong.